src/components/Resume/ResumeCreationApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+export async function postResume(resume) {
+  try {
+    await axios.post("http://localhost:7071/api/resumes", resume);
```

Comment:
Check the axios [response schema](https://axios-http.com/docs/res_schema) you can use the `statusText`. If the backend returns an error you won't get it inside the `catch`, the backend must return an error message with a status code if something went wrong. Check [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses) 
For instance:
```
{
   statusText: 'Forbidden',
   status: 403,
   data: {
      errorMessage: 'you do not have permissions'
   }
}
```

in this case although the post request failed you'll be returning a true.